### ISSUE TITLE
GEOGRAPHY/GEOMETRY database types end up as ENUM

### ISSUE THREAD

**Issue by mat813** [sequelize/sequelize-auto#445]
Given this migration:

```js
module.exports = {
  up: async (queryInterface, Sequelize) => {
      await queryInterface.createTable(
        'zones',
        {
          id: {
            allowNull: false,
            autoIncrement: true,
            primaryKey: true,
            type: Sequelize.INTEGER,
          },
          zone: {
            type: Sequelize.GEOGRAPHY('POLYGON', 4326),
            allowNull: false,
          },
        },
      );
  },
  down: async (queryInterface, _Sequelize) => {
    await queryInterface.dropTable('zones');
  },
};
```

The table ends up being just fine:

```
api_sig=> \d zones
                                         Table « public.zones »
  Colonne   |           Type           | Collationnement | NULL-able |            Par défaut             
------------+--------------------------+-----------------+-----------+-----------------------------------
 id         | integer                  |                 | not null  | nextval('zones_id_seq'::regclass)
 createdAt  | timestamp with time zone |                 | not null  | now()
 updatedAt  | timestamp with time zone |                 | not null  | now()
 zone       | geography(Polygon,4326)  |                 | not null  | 
Index :
    "zones_pkey" PRIMARY KEY, btree (id)
```

But then, sequelize-auto generates this:

```js
/* jshint indent: 2 */

const Sequelize = require('sequelize');

module.exports = function (sequelize, DataTypes) {
  return sequelize.define(
    'zones',
    {
      id: {
        autoIncrement: true,
        type: DataTypes.INTEGER,
        allowNull: false,
        primaryKey: true,
      },
      zone: {
        type: DataTypes.ENUM(),
        allowNull: false,
      },
    },
    {
      sequelize,
      tableName: 'zones',
      schema: 'public',
    }
  );
};
```

It would be great if the type of the `zone` column was correct.

**Comment by steveschmitt** [comment]
Thanks for reporting this.  What database are you using?

**Comment by mat813** [comment]
Oops, sorry, postgresql, 12.5 on arch linux, and sequelize-auto 0.7.1 with sequelize 6.3.5.

**Comment by steveschmitt** [comment]
Fixed in release 0.7.2
